satpy/etc/readers/agri_fy4b_l1.yaml

```diff
@@ -4,7 +4,11 @@
 
 reader:
   name: agri_fy4b_l1
+  short_name: AGRI FY4B L1
+  long_name: FY-4B AGRI Level 1 data HDF5 format
   description: FY-4B AGRI instrument HDF5 reader
+  status: Beta
+  supports_fsspec: false
```

Comment:
@simonrp84 @BENR0 I could be wrong, but I think any reader that is based on the HDF5 utility file handler now supports fsspec because it uses `open_file_or_filename`:

https://github.com/pytroll/satpy/blob/754e706d4ce9af4791e08342dcf4d063d461dcf1/satpy/readers/hdf5_utils.py#L46-L48

No idea if it performs very well or if h5py works well with it, but I think the support is there.

Reply:
Good point. I think this is true. I have set this to true for those readers using "hdf5_utils" but have not tested this.

Reply:
Is this something that should be tested via unittests?

Reply:
I recently added tests for the helper function:

https://github.com/pytroll/satpy/blob/e74729e53defff292c8a52376eecffadd6560bdc/satpy/tests/test_readers.py#L1207-L1231